### PR TITLE
Correct hugepeges requirement in install doc to 512 MB

### PR DESCRIPTION
**Pull request by onong**
## Description

The VPP Getting Started doc (https://projectcalico.docs.tigera.io/master/getting-started/kubernetes/vpp/getting-started) have the wrong value for the hugepages requirement, ie, 256 MB, when it should say 512 MB. Correcting it.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by lwr20 — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Comment by caseydavenport** [comment]
@onong could you raise a cherry-pick PR for the v3.22 branch as well?

**Comment by onong** [comment]
@caseydavenport 

https://github.com/projectcalico/calico/pull/5525

**Pull request merged**
